<?xml version="1.0"?>
<!-- Debug version of slam.launch -->
<launch>
    <node pkg="slam_itbdelabo" type="slam_node.py" name="slam_node" respawn="true" output="screen"></node>
    <rosparam command="load" file="$(find slam_itbdelabo)/config/slam.yaml" />
</launch>